README.md

```diff
@@ -1,13 +1,14 @@
 # OpenRCT2/OpenGraphics
 
-**OpenGraphics** is a project established to replace the base graphics of Rollercoaster Tycoon 2. The aim is to create a set of freely-useable graphics that can be used with [OpenRCT2](https://github.com/OpenRCT2/OpenRCT2), to decouple its reliance on the copyrighted material from the original game and allow anyone to play the game for free.
+**OpenGraphics** is a project established to replace the base graphics of Rollercoaster Tycoon 2. The aim is to create a set of freely-useable graphics that can be used with [OpenRCT2](https://github.com/OpenRCT2/OpenRCT2), in order to decouple its reliance on the copyrighted material from the original game, and to allow anyone to play the game for free.
 
 ## Getting Started
 
-We're in the process of setting up pipelines and deciding on the direction of OpenGraphics. If you have experience with art or tools development than consider checking out our discussions about establishing the rendering process:
+We're in the process of setting up pipelines and deciding on the direction of OpenGraphics... If you have experience with art or tool development then consider checking out our discussions about establishing the rendering process:
 https://github.com/OpenRCT2/OpenGraphics/issues/24
 
-You can already start helping out by creating 3D models for scenery objects. If possible, try to use Blender 2.7. Check out [the spreadsheet](https://drive.google.com/open?id=1ljVeYxp8ijj5z4VuSa6Xo1-_p3wACdQCRudkvkLGApc) to see what has already been made. Just mark an object as WIP, and add your name to the author field.
+You can already start helping out by creating 3D models for scenery objects. If possible, you should use Blender 2.7. Check out [the spreadsheet](https://drive.google.com/open?id=1ljVeYxp8ijj5z4VuSa6Xo1-_p3wACdQCRudkvkLGApc) to see what has already been made.
+Mark an object as WIP and add your name to the author field to contribute.
```

Comment:
Why did you choose to remove the oxford comma here, while you added one to the first paragraph?

Reply:
That's not an Oxford comma it's just grammatically incorrect.  An Oxford comma is just a comma used after the penultimate item in a list of 3 or more items.  Even still I think moving "to contribute" to be beginning of the sentence would make it sound less stilted.